packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -133,7 +128,9 @@ export default function useTabNav() {
 			// Focusing the focus capture element, which is located above and
 			// below the editor, should not scroll the page all the way up or
 			// down.
-			next.current.focus( { preventScroll: true } );
+			if ( ! isShift ) {
```

Comment:
Without this, holding down Shift+Tab will cause you to get stuck in the block since this statement would always move you forward, after focus trap.